.ado/jobs/e2e-test.yml

```diff
@@ -184,7 +184,7 @@ jobs:
               parameters:
                 buildEnvironment: ${{ parameters.BuildEnvironment }}
                 certificateName: reactUWPTestAppEncodedKey
-                buildConfiguration: Debug
```

Comment:
Does debug not work?  I'd be curious on what the issue with debug is?

Reply:
For Debug we would need to add a launch for Metro. This matches the behavior of the test app on Paper which runs in Release for automated tests.

Reply:
Really?  I suppose better than not running them.  Its possible to run debug against a prebuilt bundle just as we do in release.  But at least getting to where we are in paper is a good thing.

Reply:
I can look into that!